### PR TITLE
updates

### PR DESCRIPTION
**Pull request by KeeyanGhoreshi**
## Describe your changes

Updates to the readme(s)

### CODE REVIEW

File: PrototypeRepositoriesAndCapabilities.md [hunk 1]
```diff
@@ -36,6 +36,9 @@
       - `(1.1)(3)` Generates CDS Hooks order-sign (client) that is sent to the REMS Admin
       - `(1.2)(3)` Handles CARDS returned from REMS Admin to launch REMS SMART on FHIR App
       - Sends Rx to PIMS using NCPDP Script NewRx
+      - Communicates with REMS Admin and EHR to retrieve ETASU and prescription status
+    - Patient Portal allowing patient access to in progress forms and information about their prescriptions
```
Comment (smalho01) [inline PrototypeRepositoriesAndCapabilities.md]: typo: in-progress

File: README.md [hunk 2]
```diff
@@ -12,13 +12,13 @@ This repository contains all of the Docker, setup scripts, and documentation for
 
 ### Running the full prototype environment
 
-You can find complete end-to-end full-stack set up guides for the REMS Proof of Concept prototype at the following links:
+Complete end-to-end set up guides for the REMS Proof of Concept prototype are listed below:
 
-[Simple Set Up](SimpleSetupGuide.md) - This guide will get you up and running quickly with a demo environment for using the prototype locally. If you want to make changes or contribute to the codebase, see the detailed developer guide below.
+[Simple Set Up](SimpleSetupGuide.md) - Follow this guid to quickly setup a demo environment for local testing. If you want to make changes or contribute to the codebase, see the detailed developer guide below.
```
Comment (smalho01) [inline README.md]: typo: guide

File: SimpleSetupGuide.md [hunk 3]
```diff
@@ -84,6 +78,23 @@ See [this guide](Verify-REMS-Integration-Prototype-Works.md) to generate a test
 
 ## Cleanup and Useful Options
 
+### Obtain [Value Set Authority Center (VSAC)](https://vsac.nlm.nih.gov/) API key
```
Comment (smalho01) [inline SimpleSetupGuide.md:81]: these sections maybe should be before the Run section I think, at least the section on the VSAC key especially. I noticed before we were linking to the dev set up guide, I'm fine with having it inline instead. Also the numbering above is not consistent as we go from 1. System Requirements to 2. Run. to 4. Verify everything is working. We are missing 3 entirely, but moving the VSAC and env stuff up should fix that ordering.

**Review by smalho01 — CHANGES_REQUESTED** [review]

File: SimpleSetupGuide.md (hunk 3 above)
Comment (KeeyanGhoreshi) [reply SimpleSetupGuide.md:81]: Given that the VSAC Key is optional, I think putting it in the main workflow adds some unnecessary complexity for someone that just wants the thing to run. As of my last test, lack of a VSAC key no longer produces any errors.

**Review by smalho01 — APPROVED** [review]

**Pull request merged**